### ISSUE TITLE
AlexNet Implementation & understanding width_mult

### ISSUE THREAD

**Issue by beihaifusang** [STVIR/pysot#130]
Dear,

It is really a nice repo with clean structure and style, really appreciate the effort you guys put into it, yet I got a few questions would like to ask.

1. When comparing with original alexnet, I found there are slight differences in pySOT implementation, specifically regarding the conv stride, batchnorm instead of lrn-norm, no pooling in layer5 and using ReLU instead of P-ReLU. Is there any consideration for making those decisions?

Besides, could you please help figure out how the AlexNetLegacy different from AlexNet?

2. I actually have a hard time understanding the meaning of 'width_mult' in backbone network construction, could you please explain what it is for? It seems it is used for manipulating channel num but when and why should we do that? Is it a hyperparameter searching option? or it is done after a careful decision?

Thanks a lot.

**Comment by lb1100** [comment]
1. The modified AlexNet is similar to SiamFC, which removes some strides(or pooling layer) to make it suitable for tracking (small stride to keep spatial resolution ). And we just simply follow them and removed `group` in `conv2` and `conv4`. We believe BatchNorm is better than LRN-norm, however, we didn't do experiments on it. You can test it if you think it will be better.

2. width_mult is for doubling channel for AlexNet. It's used in `SiamRPN` (vot2018 entry). More parameters may leads to better performances.

**Comment by beihaifusang** [comment]
Thank you very much for the info. It's really helpful, regards.

**Comment by luunguyen97** [comment]
> 1. The modified AlexNet is similar to SiamFC, which removes some strides(or pooling layer) to make it suitable for tracking (small stride to keep spatial resolution ). And we just simply follow them and removed `group` in `conv2` and `conv4`. We believe BatchNorm is better than LRN-norm, however, we didn't do experiments on it. You can test it if you think it will be better.
> 2. width_mult is for doubling channel for AlexNet. It's used in `SiamRPN` (vot2018 entry). More parameters may leads to better performances.

Hi. Can you tell me how to train the Alexnet backbone after these modifications on Imagenet? I need to retrain this backbone on my custom dataset so I can't use the provided alexnet-bn in this repo.
Thanks.
